Addnewplugin.md

```diff
@@ -0,0 +1,32 @@
+# How to add e2e test for new Plugin:
+
+## Add New Tests
+
+1. All the automation code should strictly be under the src/e2e-test/.
+
+**Configuration Details required for Setup:**
+
+1. Create a maven profile in the main project for which e2e tests are required, with a dependency of e2e test framework artifact, along with your Runner file inside the failsafe plugin.
+
+NOTE: All the properties/ actions/ locators which are common to plugins, need to be or will be present in the framework code. Plugin specific properties/ actions/ locators should reside in the main project module (for eg: google-cloud).
+
+2. TestRunners and its step definition should also resides in the main project module.
+3. To maintain the service account, add the configuration for TestRunner.
```

Comment:
What does this mean? - "To maintain the service account, add the configuration for TestRunner.". 
Does the user just need to set GOOGLE_APPLICATION_CREDENTIALS environment variable? Any other configuration??

Reply:
Yes, this is the only path to specify in environment variables, apart from that, everything else is in property files.